### ISSUE TITLE
How can we train the baseline model? OE (outlier exposure) 

### ISSUE THREAD

**Issue by hyunjunChhoi** [amazon-research/long-tailed-ood-detection#1]
Hello, 
Thanks for your work. 

How can we train the baseline model such as OE (outlier exposure)

I cannot approach the similar baseline with the OE original code 

Could you help

**Comment by htwang14** [comment]
Hi, can you try the following command?

`python stage1.py --gpu 0 --ds cifar10 --Lambda 0.5 --Lambda2 0 
    --drp <where_you_store_all_your_datasets> --srp <where_to_save_the_ckpt>`

By setting `Lambda2` as `0`, we are removing the PASCL loss. By setting `Lambda` as `0.5`, we are using the OE loss. 

Let me know if you have further questions. 

Thank you!

**Comment by hyunjunChhoi** [comment]
> 

Thanks for your answer. 

I can reproduce the OE model 

However, I cannot reproduce the EnergyOE 

Accuracy of the EnergyOE is far behind from the report of paper 

How can I reproduce the EnergyOE ????(I just pasted the EnergyOE fine-tuning code and learn)

**Comment by shuaiNJU** [comment]
> Hi, can you try the following command?
> 
> `python stage1.py --gpu 0 --ds cifar10 --Lambda 0.5 --Lambda2 0 --drp <where_you_store_all_your_datasets> --srp <where_to_save_the_ckpt>`
> 
> By setting `Lambda2` as `0`, we are removing the PASCL loss. By setting `Lambda` as `0.5`, we are using the OE loss.
> 
> Let me know if you have further questions.
> 
> Thank you!

Hi, thanks to your great work. But I have recently failed to re-produce the results of OE in your paper (e.g., Texture, the 92.59% AUROC and 83.32% AUPR, CIFAR10-LT) by the provided train command and pretrained model. I got the results of:

Texture:  90.77 (AUROC), 73.99 (AUPR)

Any suggestion?
